assets/js/components/promoBanner/promo-banner.js

```diff
@@ -0,0 +1,109 @@
+            cookieData.name="S'mores";
+            cookieData.image="smores";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        case 'February':
+            cookieData.name="Adventurefuls";
+            cookieData.image="adventurefuls";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        case 'March':
+            cookieData.name="Do-si-dos";
+            cookieData.image="dosidos";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        case 'April':
+            cookieData.name="Thin Mints";
+            cookieData.image="thinmints";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        case 'May':
+            cookieData.name="Tagalongs";
+            cookieData.image="tagalongs";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        case 'June':
+            cookieData.name="Trefoils";
+            cookieData.image="trefoils";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        case 'July':
+            cookieData.name="Lemon-ups";
+            cookieData.image="lemonups";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        case 'August':
+            cookieData.name="Raspberry Rallys";
+            cookieData.image="raspberyrally";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        case 'September':
+            cookieData.name="Toffeetastic";
+            cookieData.image="toffetastic";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        case 'October':
+            cookieData.name="Samoas";
+            cookieData.image="samoas";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        case 'November':
+            cookieData.name="Adventurefuls";
+            cookieData.image="adventurefuls";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        case 'December':
+            cookieData.name="Raspberry Rallys";
+            cookieData.image="raspberryrally";
+            cookieData.description="Feeling adventurous? Try this month's signature cookie...";
+            break;
+        default:
+            cookieData.name="No cookie";
+            cookieData.image="adventurefuls";
+            cookieData.description="no cookie right now";
+            break;
+    }
+    return cookieData;
+
+}
+
+class PromoBanner extends HTMLElement {
```

Comment:
I think I just did it out of habit since you usually would create a new promo banner component in React. Would it be more preferable/efficient to use a query selector instead of calling the custom element on every page?

Reply:
No worries! I'm used to working with vanilla so I'm not used to building with components. I think to stay consistent, we can keep this implementation :)